src/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -117,12 +117,6 @@ public static void ThrowInvalidOperationException_SerializationDataExtensionProp
             throw new InvalidOperationException(SR.Format(SR.SerializationDataExtensionPropertyInvalid, jsonClassInfo.Type, jsonPropertyInfo.PropertyInfo.Name));
         }
 
-        [MethodImpl(MethodImplOptions.NoInlining)]
-        public static void ThrowInvalidOperationException_SerializationDataExtensionPropertyInvalid(JsonClassInfo jsonClassInfo, Type invalidType)
-        {
-            throw new InvalidOperationException(SR.Format(SR.SerializationDataExtensionPropertyInvalidElement, jsonClassInfo.Type, jsonClassInfo.DataExtensionProperty.PropertyInfo.Name, invalidType));
```

Comment:
Can you remove the `SerializationDataExtensionPropertyInvalidElement` from the resx file please.

Reply:
you're right thank's, done.